docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
```

Comment:
I don't like the assumption of difficulty here. Much as we shouldn't assume things are easy for people, this assumes hard for people. I'd like to this rewritten to be to not make assumption about if the reader is finding the topic easy or hard but rather address that if they are finding it hard, don't worry, they aren't alone in that. Some people catch on with reference capabilities easily, others struggle with different parts.

Reply:
I was parrotting the vibes from the intro to https://tutorial.ponylang.io/reference-capabilities/capability-matrix.html here. Now that you mention it, I actually find reference capabilities relatively intuitive, and this document was my only confusion. I'll tone it down.